s2i-gows/files/builds/gows.yml

```diff
@@ -0,0 +1,151 @@
+apiVersion: v1
```

Comment:
Should this be a template instead of a list of object?

Reply:
Thank you @sabre1041 
I was deliberating whether it should be a template or not. The first iteration was a template but with so many things that are custom I decided to hardcode things and make it a list.
Let me take another stab at a template as it would probably be the right thing:tm: to do (at least make the site you want to deploy a parameter, `demo` might not cut it for everyone ;)